.github/workflows/ci.yml

```diff
@@ -124,7 +124,7 @@ jobs:
         env:
           PHP_CS_FIXER_IGNORE_ENV: ${{ matrix.PHP_CS_FIXER_IGNORE_ENV }}
           FAST_LINT_TEST_CASES: ${{ matrix.FAST_LINT_TEST_CASES }}
-        run: vendor/bin/phpunit ${{ matrix.phpunit-flags }}
+        run: vendor/bin/phpunit tests/Fixer/Basic/NonPrintableCharacterFixerTest.php ${{ matrix.phpunit-flags }}
```

Comment:
is there a case when someone is using zero-width or non-breaking space in code *by purpose* ?

Reply:
I think both linked examples (from the description) are made by purpose.

Reply:
1st example - i don't see usage of `zero-width space` just by looking at it. it's not obvious to me where it is or how it's needed

2nd example - way, way, way too big edge case

Reply:
I agree with @keradus, I don't think the zero-width space in the first example is intended. I do love the second example (currently advocating to allow french diacritics in code at work, I will use it 😆️) but yes, it's really an edge case, I would suggest disabling the rule.

Reply:
> 1st example - i don't see usage of `zero-width space` just by looking at it. it's not obvious to me where it is or how it's needed

@keradus it's exactly the same as in the example in the description - between `*` and `/` - and I believe it is intended because if there was nothing there that would be broken PHP syntax as it would close the PHPDoc.

> 2nd example - way, way, way too big edge case

I agree that this is extreme edge case, but still, do we want PHP CS Fixer to produce invalid syntax in such case?

Reply:
:+1: